source/catalog.html

```diff
@@ -0,0 +1,216 @@
+              <dt>Вкус</dt>
+              <dd>Рыба</dd>
+              <dt>Цена</dt>
+              <dd>1000 Р.</dd>
+            </dl>
+          </div>
+        </div>
+        <a href="#" role="button">Заказать</a>
+      </li>
+      <li>
+        <div>
+          <img src="#" alt="Изображение коробки Cat Energy Slim, вес 500 грамм, вкус - Гречка.">
+          <div>
+            <h3><a href="#" id="500-slim">Cat Energy Slim 500 г</a></h3>
+            <dl>
+              <dt>Объем</dt>
+              <dd>500 г.</dd>
+              <dt>Вкус</dt>
+              <dd>Гречка</dd>
+              <dt>Цена</dt>
+              <dd>400 Р.</dd>
+            </dl>
+          </div>
+        </div>
+        <a href="#" role="button">Заказать</a>
+      </li>
+      <li>
+        <div>
+          <img src="#" alt="Изображение коробки Cat Energy Slim, вес 1000 грамм, вкус - Гречка.">
+          <div>
+            <h3><a href="#">Cat Energy Slim 1000 г</a></h3>
+            <dl>
+              <dt>Объем</dt>
+              <dd>1000 г.</dd>
+              <dt>Вкус</dt>
+              <dd>Гречка</dd>
+              <dt>Цена</dt>
+              <dd>700 Р.</dd>
+            </dl>
+          </div>
+        </div>
+        <a href="#" role="button">Заказать</a>
+      </li>
+      <li>
+        <div>
+          <img src="#" alt="Изображение коробки Cat Energy Slim, вес 500 грамм, вкус - Рис.">
+          <div>
+            <h3><a href="#">Cat Energy Slim 500 г</a></h3>
+            <dl>
+              <dt>Объем</dt>
+              <dd>500 г.</dd>
+              <dt>Вкус</dt>
+              <dd>Рис</dd>
+              <dt>Цена</dt>
+              <dd>500 Р.</dd>
+            </dl>
+          </div>
+        </div>
+        <a href="#" role="button">Заказать</a>
+      </li>
+      <li>
+        <h3><a href="#">Показать еще 100500 товаров</a></h3>
+        <p>На самом деле вкусов гораздо больше!</p>
+        <a href="#" role="button">Показать все</a>
+      </li>
+    </ul>
+  </section>
+  <section>
+    <h2>Дополнительные товары</h2>
+    <ul>
+      <li>
+        <h3><a href="#">Сахарозаменитель</a></h3>
+        <dl>
+          <dt>1 упаковка (100 г)</dt>
+          <dd>200 Р.</dd>
+        </dl>
+        <a href="#" role="button">Заказать</a>
+      </li>
+      <li>
+        <h3><a href="#">Питьевая вода</a></h3>
+        <dl>
+          <dt>5 литров</dt>
+          <dd>50 Р.</dd>
+        </dl>
+        <a href="#" role="button">Заказать</a>
+      </li>
+      <li>
+        <h3><a href="#">Молоко</a></h3>
+        <dl>
+          <dt>1 литр</dt>
+          <dd>100 Р.</dd>
+        </dl>
+        <a href="#" role="button">Заказать</a>
+      </li>
+      <li>
+        <h3><a href="#">Витамины</a></h3>
+        <dl>
+          <dt>1 упаковка (30 г)</dt>
+          <dd>300 Р.</dd>
+        </dl>
+        <a href="#" role="button">Заказать</a>
+      </li>
+    </ul>
+    <div>Закажите все и получите чехол для кота в подарок!</div>
```

Comment:
```suggestion
    <div><p>Закажите все и получите чехол для кота в подарок!</p></div>
```
текст есть текст, див то нам нужен только для оформления бекграунда, текст пусть будет в своем фразовом теге